build/conf.js

```diff
@@ -47,7 +47,11 @@ export default {
     /**
      * Port number of the development server started by Gulp.
      */
-    devServerPort: 8080,
+    devServerPort: 9090,
```

Comment:
You should change this default in the dashboard.go too. To have consistent default ports here and when you start application manually.

Other option is to pick here a random free port. This would allow us to run two `gulp serve` tasks at the same time. What do you think? Is there a nodejs api for picking random free port?